stable/keel/values.yaml

```diff
@@ -7,6 +7,26 @@ image:
   tag: 0.9.5
   pullPolicy: IfNotPresent
 
+## Additional environment variables to set
+extraEnvs: []
+# extraEnvs:
+#   - name: FOO
+#     valueFrom:
+#       secretKeyRef:
+#         key: FOO
+#         name: secret-resource
+
+rbac:
```

Comment:
You can remove this, since you only add a service account.

Reply:
Actually, I wanted to add the extra var support to avoid having to sync with upstream and PR here each time an ENV is added.